src/toast/unported/tests/ops_dipole.py

```diff
@@ -149,13 +149,28 @@ def test_sim(self):
         # construct distributed maps to store the covariance,
         # noise weighted map, and hits
 
-        invnpp = DistPixels(self.data, nnz=1, dtype=np.float64, nest=False,)
+        invnpp = DistPixels(
```

Comment:
`black` seems to have a hard time deciding how to format these lines.

Reply:
Yes, not a fan of what it did to some of the code...

Reply:
The challenge is that black defaults change slightly between versions, and we are all using different versions.  I have just updated the version in the latest cmbenv stack at nersc, and have installed the latest version locally.  It is slightly painful, but I'll just remind everyone the utter formatting hell that we went through in the recent past.  I feel these annoyances are much better than inconsistency everywhere...